leetcode/generate_md.py

```diff
@@ -1,5 +1,18 @@
-print('hello world!')
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 it`s working! 
 Mistakes: 
 Dude...it is not even in right format.